stdlib/3/builtins.pyi

```diff
@@ -38,9 +38,26 @@ class object:
     def __setattr__(self, name: str, value: Any) -> None: ...
     def __eq__(self, o: object) -> bool: ...
     def __ne__(self, o: object) -> bool: ...
+    def __lt__(self, o: object) -> bool: ...
```

Comment:
These 4 definitions will break mypy. It's best to leave them out and potentially file a separate issue about these.